sunpy/net/_attrs.py

```diff
@@ -64,7 +64,7 @@ def collides(self, other):
         # Use exact type checking here, because otherwise it collides with all
         # subclasses of itself which can have completely different search
         # meanings.
-        return type(other) is type(self)
+        return isinstance(other, type(self))
```

Comment:
Doesn't the comment above imply that this should stay as `is` and not `istinstance`?

Reply:
Yes, I will need to fix that.